llvm/include/llvm/IR/CallingConv.h

```diff
@@ -287,6 +287,18 @@ namespace CallingConv {
     // Calling convention for AMDGPU whole wave functions.
     AMDGPU_Gfx_WholeWave = 124,
 
+    /// CHERIoT_CompartmentCall - Calling convention used for CHERI when
```

Comment:
```suggestion
    /// Calling convention used for CHERI when
```
etc. No need to repeat the enum name in the doc comment.